dist/index.tsx

```diff
@@ -11,6 +11,8 @@ const HALF_RAD = Math.PI/2
 export default class AnimateNumber extends Component {
 
   props: {
+    // Text Component의 최대 폰트 크기 배율 설정
+    maxFontSizeMultiplier?: number | null | undefined,
```

Comment:
```suggestion
    maxFontSizeMultiplier?: number
```

Why did you use `number | null | undefined`?

Reply:
![image](https://github.com/JeffGuKang/react-native-animate-number/assets/20368822/b2d7ca5e-66d2-47a7-9310-89d454afd03b)

that type is RN Text Component maxFontSizeMultiplier type

AnimateNumber Component just pass that prop into Text Component
so i just use Text Component maxFontSizeMultiplier type